plugins/cache.rst

```diff
@@ -33,22 +33,34 @@ control headers from the server as specified in :rfc:`7234`. It needs a
         [$cachePlugin]
     );
 
+.. note::
+
+    Since v1.3.0 does the ``CachePlugin`` have 2 factory methods to easily setup the plugin by caching type.
+
+    .. code-block:: php
+
+        $cachePlugin = CachePlugin::clientCache($pool, $streamFactory, $options); // Which allows caching responses with the 'private' or 'no-store' cache directive
```

Comment:
I don't know if it works as the preview screen on github doesn't show the output good. I will change the how the comments are placed!

I could also put a not and then in a separate code block (outside of the note) the example code?